request_a_govuk_domain/request/templatetags/admin_tags.py

```diff
@@ -15,22 +17,49 @@ def changed_fields(obj):
     if obj.prev_record:
         delta = obj.diff_against(obj.prev_record)
         changes = list(filter(lambda x: x != "last_updated_by", delta.changed_fields))
-        return changes if changes else "Updated through review"
+        return changes if changes else "--"
```

Comment:
Can you put something more explicit than `--` ? What does it mean for the user when there are no changes? Why show entries in the history if there are no changes?